src/test/java/de/zalando/aruha/nakadi/repository/kafka/KafkaPartitionsCalculatorTest.java

```diff
@@ -0,0 +1,128 @@
+package de.zalando.aruha.nakadi.repository.kafka;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.config.JsonConfig;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+import static org.hamcrest.Matchers.notNullValue;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KafkaPartitionsCalculatorTest {
+
+    private static final String TEST_DATA = "[" +
```

Comment:
I would suggest to move this to a separate json-file. (I think we already have json-files for some of the tests)
Json inside Java string looks really ugly :)